### DISCUSSION TITLE
Speech Setup Navigation

### DISCUSSION THREAD

**Discussion by SF4User** [devemux86/cruiser#186] [Q&A] [answered]
I now had some problems with the Basic Mobile Phone Setup. . . Data rooming, internet data sharing, voice output. . .
That's not the issue here - solved. . . But I searched for a long time until I found out that or whether the navigation voice output works in the Cruiser.
Google Maps, Calimoto, Kurviger. . . all give a 1st voice output right after the start.
So you knew immediately after the start whether the voice output was working or whether the voice output was also available via Bluetooth on the helmet set.
With the Cruiser, there is no voice output when the navigation starts. . . you only find out at the next turning point whether the voice output is actually there.
Isn't it possible to send a first voice output when starting the navigation? like the other Navi's too?

**Comment by devemux86** [comment, marked as answer]
@SF4User thanks for the report!

Cruiser 3.0.20 offers a "Voice test" button in navigation options (long press top-left navigation panel).

"Voice test" button remains on screen, we can press it many times testing the sound on our devices.
Does it help with your voice tests?

More details can also be found here:
- https://github.com/devemux86/cruiser/discussions/106

<img src="https://github.com/devemux86/cruiser/assets/3484020/71f06042-026a-4e2d-af8c-f4efacdc8184" width="50%"/>

**Reply by SF4User** [reply]
Perfect! Thanks
learned something again . . . :)

**Comment by 0709wiwiwi** [comment]
> Isn't it possible to send a first voice output when starting the navigation? like the other Navi's too?> 

https://github.com/devemux86/cruiser/discussions/106#discussioncomment-5928016
